src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -268,15 +268,33 @@
             </itemOperation>
 
             <itemOperation name="shop_remove_item">
-                <attribute name="method">PATCH</attribute>
-                <attribute name="path">/shop/orders/{id}/remove</attribute>
+                <attribute name="method">DELETE</attribute>
+                <attribute name="path">/shop/orders/{id}/items/{itemId}</attribute>
                 <attribute name="messenger">input</attribute>
-                <attribute name="input">Sylius\Bundle\ApiBundle\Command\Cart\RemoveItemFromCart</attribute>
+                <attribute name="controller">Sylius\Bundle\ApiBundle\Controller\DeleteOrderItemAction</attribute>
+                <attribute name="write">false</attribute>
```

Comment:
Is it needed?

Reply:
yes, without it, after job from the controller, Api-platform will delete order (cart in this case).